resource_packs/files/peace_and_quiet/quieter_water/sounds.json

```diff
@@ -27,6 +27,9 @@
 			},
 			"cauldron_drip.water.pointed_dripstone": {
 				"volume": 0.1
+			},
+			"drip.water.pointed_dripstone" : {
+			   "pitch" : 0.1
```

Comment:
Why are you trying to decrease**pitch**? **volume** should be decreased here as pitch is completely different thing and it will make the sound different instead of quieter

Reply:
I kept the wrong key, wasnt looking tbh